msteams-platform/webhooks-and-connectors/what-are-webhooks-and-connectors.md

```diff
@@ -26,6 +26,27 @@ Connectors allow users to subscribe to receive notifications and messages from t
 
 Incoming Webhooks help in posting messages from apps to Teams. If Incoming Webhooks are enabled for a team in any channel, it exposes the HTTPS endpoint, which accepts correctly formatted JSON and inserts the messages into that channel. For example, you can create an Incoming Webhook in your DevOps channel, configure your build, and simultaneously deploy and monitor services to send alerts.
 
+#### Notification bot or incoming webhook - choose the right one!
+
+Before you start to learn how to build Incoming webhooks, you may also want to know that you can build Notification Bot using Teams Toolkit. Notification Bots can enable more customizable experience to meet different business scenarios.
+
+Learn more about the differences between Notification Bot and Incoming webhook so that you can choose correct solutions for your scenarios:
+
+| &nbsp; | Notification bot |  Incoming Webhook |
+| --- | --- | --- |
+| What is it? | A Teams app | A Teams feature |
+| Installation required | Yes | No |
+| Suitable scenarios | • Receive regular notifications and messages periodically, for example, receive daily notification of team tasks. <br>  • Receive notifications and messages based on real events. For example, once teammates upload files, you receive notifications. | Communicate with external apps and receive notifications and messages from other apps. |
+| Scope configuration | • Teams channel <br> • Group chat <br> • Personal chat | Teams channel |
+| Message process | A Notification Bot works as a Teams application. You can define your business logic to process data and show data in a customized format. | Webhook is a Teams feature rather than a Teams application, so it only receives and shows data without processing. |
+| Retrieve Teams context | Notification Bot can retrieve Teams context such as the channel or user information, messages, etc. | No |
+| Send Adaptive Card | Yes | Yes |
+| Send a welcome message | Can send a welcome message | No welcome message |
+| Trigger Supported | All triggers supported. If you use Teams Toolkit, you can quickly get template projects with following triggers: <br> • Time trigger hosted on Azure functions. <br> • Restify HTTP trigger hosted on Azure app service <br> • HTTP trigger hosted on Azure Functions | All triggers supported |
+| Building Tools | • [Teams Toolkit for Visual Studio Code](/toolkit/teams-toolkit-fundamentals) <br> • [Teams Toolkit for Visual Studio](/toolkit/teams-toolkit-overview-visual-studio) <br> • [Teams Toolkit CLI](/toolkit/teamsfx-cli) <br> • [TeamsFx SDK](/toolkit/teamsfx-sdk) | No tools required |
```

Comment:
The links of "Building Tools" in comparison table don't work. They can't link to the right pages. Please help with fix it.

Reply:
Some links don't work in review mode, but they do in live site. Even so, I'll add the links again.